paper-fab-speed-dial.html

```diff
@@ -71,6 +71,11 @@
 	Polymer({
 		is: 'paper-fab-speed-dial',
 		properties: {
+			icon: {
+				type: String,
+				notify: true,
```

Comment:
Why should the `icon` property notify on changes? My understanding is that the user can set the icon but that `paper-fab-speed-dial` itself isn't expected to change the icon?
In case there is no reason for `notify`: Could you please remove it as it will keep the API cleaner.
Thanks!